### PR TITLE
Trying stuff

### PR DESCRIPTION
**Pull request by tcober**

### CODE REVIEW

**Review by experimatt — COMMENTED** [review]
I think you should remove the axios stuff, we should merge my PR (#5) in, and then merge master into your branch to reconcile any other conflicts. Otherwise, everything in this PR looks fine, except I think we'll need to address our local environment node/yarn/env differences sometime soon, and agree on one set.

**Comment by tcober** [comment]
@experimatt @jenessawhite 
Sure! I'll do that tonight. What do you think of having a branch out there with my solution just in case that Gridsome third party stuff gets weird? I hope you don't think I'm not respecting how much work you guys put in! I just am little weary of some of that Gridsome plugin stuff since everything is still pretty early in it's development.

**Comment by experimatt** [comment]
Sure - it doesn't hurt anything to leave your branch out there, as long as we come back and clean it up later if we never use it

**Comment by experimatt** [comment]
You've got conflicts now that I've merged my PR in -- should be easy to separate out :)

**Review by jenessawhite — APPROVED** [review]

File: gridsome.server.js [hunk 1]
```diff
@@ -5,12 +5,26 @@
...
-    // Use the Data Store API here: https://gridsome.org/docs/data-store-api
-  })
+const axios = require("axios");
 
-  api.createPages(({ createPage }) => {
-    // Use the Pages API here: https://gridsome.org/docs/pages-api
-  })
-}
+module.exports = function(api) {
+  api.loadSource(async store => {
```
Comment (experimatt) [inline gridsome.server.js:11]: Is this going to break things on the server? I presume everything in `gridsome.config.js` is all we need for airtable, and that we can remove this part

Comment (tcober) [reply gridsome.server.js:11]: @experimatt It does not break things on the server. Do you have the Netlify login creds? But yes go ahead and remove it next time you get a chance. I will not get a chance to touch any of this til late in the weekend.

Comment (experimatt) [reply gridsome.server.js:11]: It's OK I haven't had a chance to look at anything since we did our hacking weekend two weeks agin 🙈

**Pull request merged**